.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -4989,107 +4989,163 @@ stages:
   variables:
     targetShaId: $[ stageDependencies.merge_commit_id.fetch.outputs['set_sha.sha']]
     targetBranch: $[ stageDependencies.merge_commit_id.fetch.outputs['set_sha.branch']]
+    scenarioGroups: "appsec"
+    additionalScenarios: '[\"PARAMETRIC\", \"REMOTE_CONFIG_SCENARIOS\", \"TELEMETRY_SCENARIOS\", \"DEFAULT\", \"INTEGRATIONS\", \"INTEGRATIONS_AWS\", \"CROSSED_TRACING_LIBRARIES\", \"DEBUGGER_SCENARIOS\", \"+S PROFILING +S TRACE_PROPAGATION_STYLE_W3C +S LIBRARY_CONF_CUSTOM_HEADER_TAGS\"]'
   jobs:
-  - template: steps/update-github-status-jobs.yml
-    parameters:
-      jobs: [test]
+    - template: steps/update-github-status-jobs.yml
+      parameters:
+        jobs: [compute_scenarios, tests]
+        
+    - job: compute_scenarios
+      timeoutInMinutes: 60
+      displayName: Compute system-tests scenarios
+      pool:
+          vmImage: ubuntu-latest
+          
+      steps:
+        - checkout: none
+          
+        - task: UsePythonVersion@0
+          inputs:
+            versionSpec: '3.12'
+          displayName: Install python 3.12
 
-  - job: test
-    timeoutInMinutes: 60
-    pool:
-      vmImage: ubuntu-latest
+        - script: git clone --depth 1 https://github.com/DataDog/system-tests.git
+          displayName: Get system tests repo
+          
+        - script: |
+            ./build.sh -i runner
+          displayName: Install runner
+          workingDirectory: system-tests
+          
+        - script: |
+            set -e
+            
+            source venv/bin/activate
+            PYTHONPATH=. python utils/scripts/compute-workflow-parameters.py dotnet -g $(scenarioGroups) > out.txt
```

Comment:
In a world where I'm desperately trying to _reduce_ the amount of code we have embedded in the yaml, this makes me sad 😅 

But I don't think it makes sense to do it anywhere else (e.g. in Nuke etc) 🤷‍♂️

Reply:
Yes sorry for new embed code 😬 
We could in the future ask to include this code inside the system-tests repo so the `compute-workflow-parameters.py` can output directly what we want for Azure Devops with the given parameters.

Reply:
Yeah, it might make sense to generalize this in system tests at some point, especially if other CI systems can use it too 🤔 I think it's fine to leave it here for now though 🙂